lib/target/sol20/classic/sol20_crt0.asm

```diff
@@ -41,7 +41,9 @@
 	defc	CONSOLE_COLUMNS = 64
         INCLUDE "crt/classic/crt_rules.inc"
 
+     IFNDEF CRT_ORG_CODE
```

Comment:
It looks like we normally do this with:

```
IF !DEFINED_CRT_ORG_CODE
```

Not that it really matters either way...

Reply:
ok I've changed it to `IF !DEFINED_CRT_ORG_CODE`